lib/conversation/replies.js

```diff
@@ -0,0 +1,148 @@
+'use strict';
+
+/* eslint-disable no-param-reassign */
+
+const logger = require('winston');
+
+const Command = require('./command');
+const actions = require('./actions');
+const helpers = require('../helpers');
+const stathat = require('../stathat');
+const BotRequest = require('../../app/models/BotRequest');
+
+function customMsgDecorator(fn) {
+  return (args) => {
+    fn(args);
+    return helpers.sendResponse(args.res, 200, args.text);
+  };
+}
+
+function renderEndConversationMsg(fn) {
+  return (type, args) => helpers.renderMessageForMessageType(args.req, type)
+      .then((replyMessage) => {
+        args.text = replyMessage;
+        fn(args);
+        args.req.user.postDashbotOutgoing(args.text);
+        stathat.postStat(`campaignbot:${args.text}`);
+        BotRequest.log(args.req, 'campaignbot', type, args.text);
+        return helpers.sendResponse(args.res, 200, args.text);
+      })
+      .catch(err => helpers.sendErrorResponse(args.res, err));
+}
+
+function renderContinueConversationMsg(fn) {
+  return (type, args) => helpers.renderMessageForMessageType(args.req, type)
```

Comment:
What do you think about renaming `type` to `messageType`? I think it would help with readability. 

`messageTemplate` could also be a contender... I defined this property as [`template` in the Slothie `Message` model](https://github.com/DoSomething/slothie/blob/264a3a8df76bae44240e039eb0884128de891538/app/models/Message.js#L18), inspired how Dashbot lets you pass an optional [Template ID](https://www.dashbot.io/sdk/template) in their message analytics.